pkgs/development/compilers/bs-platform/default.nix

```
@@ -4,14 +4,14 @@ let
 in
 (build-bs-platform {
   inherit stdenv runCommand fetchFromGitHub ninja nodejs python3;
-  version = "7.0.1";
+  version = "7.1.0";
   ocaml-version = "4.06.1";
 
   src = fetchFromGitHub {
     owner = "BuckleScript";
     repo = "bucklescript";
-    rev = "52770839e293ade2bcf187f2639000ca0a9a1d46";
-    sha256 = "0s7g2zfhshsilv9zyp0246bypg34d294z27alpwz03ws9608yr7k";
+    rev = "a7f482243fabb38292b81b7bae9cefcb8076237f";
```

revert to old commit:

```suggestion
    rev = "52770839e293ade2bcf187f2639000ca0a9a1d46";
```